src/utils/constantsV3.ts

```diff
@@ -1,11 +1,13 @@
 import { Address } from 'viem';
 import { ChainId } from './constants';
+import { balancerV3Contracts } from './balancerV3Contracts';
 
+const { Vault } = balancerV3Contracts;
 // V3
 export const VAULT_V3: Record<number, Address> = {
-    [ChainId.SEPOLIA]: '0xbA1333333333a1BA1108E8412f11850A5C319bA9',
-    [ChainId.MAINNET]: '0xbA1333333333a1BA1108E8412f11850A5C319bA9',
-    [ChainId.GNOSIS_CHAIN]: '0xbA1333333333a1BA1108E8412f11850A5C319bA9',
+    [ChainId.SEPOLIA]: Vault[ChainId.SEPOLIA],
+    [ChainId.MAINNET]: Vault[ChainId.MAINNET],
+    [ChainId.GNOSIS_CHAIN]: Vault[ChainId.GNOSIS_CHAIN],
```

Comment:
We could do this but I kinda like abandoning the `constantsV3.ts` file entirely since all ACTIVE v3 addresses will be in `balancerV3Contracts`